### PR TITLE
Revamp Seating App

### PR DESCRIPTION
**Pull request by itsnshaw**
Front end changes mostly.

### CODE REVIEW

File: server/models.py [hunk 1]
```diff
@@ -99,15 +99,57 @@ class SeatAssignment(db.Model):
...
+        name='midterm1', 
+        display_name='Midterm 1',   
+    ),  
+    Exam(   
+        offering='cal/test/fa18',  
+        name='midterm2', 
+        display_name='Midterm 2',   
+    ),  
+    Exam(   
+        offering='cal/test/fa18',
```
Comment (mehrdadn) [inline server/models.py]: Do we want to factor these out? They appear in multiple places.

Comment (itsnshaw) [reply server/models.py]: (I'm not done lol)

Comment (mehrdadn) [reply server/models.py]: Oh lol sorry!

**Comment by itsnshaw** [comment]
@mehrdadn Yeah that's one thing I'm not sure what to do. That OK course is just a test course and I just filled up the exam table. Not sure how else to set up a development environment sort of thing

**Comment by itsnshaw** [comment]
https://github.com/Cal-CS-61A-Staff/oh-queue/blob/master/config.py the office hour queue seems to do something similar, so that's kind of what I'm going for but idk 😬😬😬

**Comment by mehrdadn** [comment]
Oh I was only wondering if you could factor out the URL into a variable since I saw the same URL in multiple places. Not sure if it needs to be changed at a higher level or not. Once thing I would note though is that using `__file__` as a test path is handy but be careful it wouldn't work so well if you e.g. `pip install` this as a package at some point. Not sure if that's a concern but something to keep in mind.

**Comment by itsnshaw** [comment]
oohhh very smart. I will add that

**Pull request merged**